test/app_test_case.py

```diff
@@ -65,6 +120,7 @@ def base_url(self):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
+        TestKeyManager.generate_test_keys()
```

Comment:
Why do we need to do this for every test case? Aren't the keys only needed for the authentication test cases?

Reply:
No, we don't. I'll change this into making the keys on demand.

Reply:
In the next set of commits, the test keys will be generated just right before either one of the keys are requested.